src/storage/redis_metadata.h

```diff
@@ -197,3 +199,33 @@ class StreamMetadata : public Metadata {
   void Encode(std::string *dst) override;
   rocksdb::Status Decode(const std::string &bytes) override;
 };
+
+class SBChainMetadata : public Metadata {
+ public:
+  uint16_t n_filters;
+  uint16_t options;
+  uint16_t growth;
+
+  explicit SBChainMetadata(bool generate_version = true) : Metadata(kRedisBloomFilter, generate_version) {}
+
+  void Encode(std::string *dst) override;
+  rocksdb::Status Decode(const std::string &bytes) override;
+};
+
+class BFMetadata : public Metadata {
+ public:
```

Comment:
As a simple implemention. We can use an aligned bloom filter. And keep only:

```C
class BFMetadata : public Metadata {
 public:
  uint32_t hashes;
  uint64_t entries;

  double error;
  double bpe;

  uint64_t bf_bytes;
};
```

We can even implement it easier using a BlockSplitBloomFilter.